values.yaml

```diff
@@ -379,9 +379,16 @@ ingress:
   hosts:
     - host: chart-example.local
       paths:
+        # You can manually specify the service name and service port if required.
+        # (For exemple if you're using AWS ALB ingress controller and want to set up automatic SSL redirect (https://kubernetes-sigs.github.io/aws-alb-ingress-controller/guide/tasks/ssl_redirect/#redirect-traffic-from-http-to-https))
+        # - path: /*
+        #  serviceName: ssl-redirect
+        #  servicePort: use-annotation
+        
+        # Or you can let the template to set it for you.
+        # Both rules can be combined.
         # NB: You may also want to set the basePath above
-        - /
-
+        - path: /
```

Comment:
I think I'd rather keep the default as it was, especially given my suggestions above.

Reply:
Yes, it shouldn't be commit as this, sorry for that